### PR TITLE
Show explanation if user cannot add new email

### PR DESCRIPTION
**Pull request by n0toose**
This change concerns the "Manage Email Addresses" submenu
in the Account settings.

### CODE REVIEW

**Review by delvh — APPROVED** [review]

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -688,6 +688,7 @@ emails = Email Addresses
 manage_emails = Manage Email Addresses
 manage_themes = Select default theme
 manage_openid = Manage OpenID Addresses
+email_cant_add = You cannot add any new email addresses at this time. Please try again later.
```
Comment (techknowlogick) [inline options/locale/locale_en-US.ini:691]: Could you remove the "try again later"? As that may create confusion for the user, and they'll try to refresh the page without knowing why they can't add a new email. From the ticket you linked it sounds like the user hadn't verified their email yet.

**Review by techknowlogick — CHANGES_REQUESTED** [review]

**Comment by wxiaoguang** [comment]
I do not think it is right to merge some "unclear" changes into code base.

I will figure out this problem by a new PR.

**Comment by wxiaoguang** [comment]
I think I understand the problem now.

->  Clarify the reason why the user can't add a new email if there is a pending activation #25509

**Comment by techknowlogick** [comment]
Thanks for this PR. I will close it in favour of the other PR.

**Pull request closed without merging**